### PR TITLE
EAR-1170, EAR-1171, EAR-1173: Various style fixes for folders.

### PR DESCRIPTION
**Pull request by MattElcock**
### What is the context of this PR?

This PR fixes the following style problems:

- EAR-1170 - The section nav item truncation is too long. It causes the menu to be scrollable to the right.
- EAR-1171 - Question title now bold in left nav
- EAR-1173 - Titles of questions too close to errors

### How to review

Ensure those style problems are fixed.

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process

### CODE REVIEW

File: eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/index.js [hunk 1]
```diff
@@ -60,22 +60,22 @@ const OpenAllSectionsBtn = styled(Button).attrs({
   variant: "tertiary-light",
   small: true,
 })`
-  margin: 0.8em 0 0.425em 2.2em;
```
Comment (jasehumphr) [inline eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/index.js:63]: https://www.w3schools.com/css/css_margin.asp What was wring with the shorthand?

File: eq-author/src/components/Badge/index.js [hunk 2]
```diff
@@ -38,14 +38,14 @@ export const smallBadge = css`
...
   border-radius: 0.7em;
-  color: white;
   line-height: 1;
   margin-left: auto;
   font-size: 0.9rem;
 `;
 
 const Badge = styled.span`
-  background-color: ${colors.red};
+  background-color: ${colors.red} !important;
```
Comment (jasehumphr) [inline eq-author/src/components/Badge/index.js:47]: You are lucky the linter hasn't pulled you up on the !important

Comment (MattElcock) [reply eq-author/src/components/Badge/index.js:47]: That was on purpose; the badge should always have this style but I needed to do a ` > span` elsewhere, which effected this.

Comment (jasehumphr) [reply eq-author/src/components/Badge/index.js:47]: Sometimes the linter doesn't allow. I love using !important but have had my knuckles wrapped

File: eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/index.js (hunk 1 above)
Comment (MattElcock) [reply eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/index.js:63]: Nothing, I just didn't use it. I've swapped back to shorthand.

**Review by jasehumphr — APPROVED** [review]

**Comment by tjamesmac** [comment]
need to check something a sec before someone merges this

File: eq-author/src/components/Badge/index.js [hunk 3]
```diff
@@ -39,14 +39,14 @@ export const smallBadge = css`
...
```
Comment (tjamesmac) [inline eq-author/src/components/Badge/index.js:48]:
```suggestion
  background-color: ${colors.red};
```

File: eq-author/src/components/Badge/index.js [hunk 4]
```diff
@@ -39,14 +39,14 @@ export const smallBadge = css`
...
+  color: ${colors.white} !important;
```
Comment (tjamesmac) [inline eq-author/src/components/Badge/index.js:49]:
```suggestion
  color: ${colors.white};
```

File: eq-author/src/components/CollapsibleNavItem/index.js [hunk 5]
```diff
@@ -60,25 +59,36 @@ const Title = styled.button`
   &:disabled {
     background: ${colors.orange};
     outline: none;
-    color: ${colors.black};
 
-    svg {
+    > span {
+      color: ${colors.black};
+    }
```
Comment (tjamesmac) [inline eq-author/src/components/CollapsibleNavItem/index.js:65]:
```suggestion
    ${Title} {
      color: ${colors.black};
    }
```
Just need to move title above the button then

File: eq-author/src/components/NavItem/index.js [hunk 6]
```diff
@@ -39,9 +33,12 @@ const Title = styled.button`
   &:disabled {
     background: ${colors.orange};
     outline: none;
-    color: ${colors.black};
 
-    svg {
+    > span {
+      color: ${colors.black};
+    }
```
Comment (tjamesmac) [inline eq-author/src/components/NavItem/index.js:39]:
```suggestion
    ${Title} {
      color: ${colors.black};
    }
```

File: eq-author/src/components/NavItem/index.js [hunk 7]
```diff
@@ -58,8 +55,12 @@ const Title = styled.button`
   }
 `;
 
-const StyledTruncated = styled(Truncated)`
+const Title = styled(Truncated)`
+  color: ${colors.white};
+  flex: 0 1 75%;
+  margin-right: 0.5em;
```
Comment (tjamesmac) [inline eq-author/src/components/NavItem/index.js:61]: Move this above `Button`

**Review by tjamesmac — CHANGES_REQUESTED** [review]

**Comment by tjamesmac** [comment]
The introduction and section titles are no longer bold

**Review by martyncolmer — APPROVED** [review]

**Review by jasehumphr — APPROVED** [review]

**Pull request merged**